src/processSymbolLayer.ts

```diff
@@ -622,11 +622,11 @@ const processSymbolHatchFill = (layer: SymbolLayer): Symbolizer[] => {
 
   // Geoserver acts weird with tilted lines. Empirically, that's the best result so far:
   // Takes the double of the raw separation value. Line and dash lines are treated equally and are looking good.
+  // For the straight hatch markers, we need to divide the value by 2 because in the sldParser it will be multiplied by 2.
```

Comment:
```suggestion
  // For the straight hatch markers, it looks that dividing the value by 2 gives best results. 
```
That's less explicit, but in geostyler parser, we don't address our code to a specific second parser. Potentially, peoples will read lyrx and write it in mapbox style.

If that makes a big issue later, we can add an option in the constructor to adapt sizes only in some cases.